tailwind.config.js

```diff
@@ -5,19 +5,33 @@ module.exports = {
   content: ['./pages/**/*.tsx', './components/**/*.tsx'],
   theme: {
     colors: {
-      blue600: '#1E88E5',
-      blue700: '#1976D2',
-      grey900: '#212529',
-      grey800: '#343A40',
-      grey700: '#495057',
-      grey650: '#6B737B',
-      grey600: '#6C757D',
-      grey500: '#ADB5BD',
-      grey400: '#CED4DA',
-      grey300: '#DEE2E6',
-      grey200: '#E9ECEF',
-      grey100: '#F8F9FA',
-      red200: '#EF9A9A',
+      /* light themes */
+      blue600: 'rgba(var(--blue600))',
+      blue700: 'rgba(var(--blue700))',
+      grey900: 'rgba(var(--grey900))',
+      grey800: 'rgba(var(--grey800))',
+      grey700: 'rgba(var(--grey700))',
+      grey650: 'rgba(var(--grey650))',
+      grey600: 'rgba(var(--grey600))',
+      grey500: 'rgba(var(--grey500))',
+      grey400: 'rgba(var(--grey400))',
+      grey300: 'rgba(var(--grey300))',
+      grey200: 'rgba(var(--grey200))',
+      grey100: 'rgba(var(--grey100))',
+      red200: 'rgba(var(--red200))',
+
+      /* dark themes */
+      hyperlink: 'rgba(var(--hyperlinkDark))',
+      backgroundBlueDark: 'rgba(var(--backgroundBlueDark))',
+      LabelHeadingsDark: 'rgba(var(--LabelHeadingsDark))',
+      grey650Dark: 'rgba(var(--grey650Dark))',
+      globalBorderGrey: 'rgba(var(--globalBorderGrey))',
+      globalBackgroundGrey: 'rgba(var(--grey300Dark))',
+      boxscoreBorderGrey: 'rgba(var(--boxscoreBorderGrey))',
+      backgroundGrey100: 'rgba(var(--backgroundGrey100))',
+      white: `rgba(var(--white))`,
```

Comment:
In here we should use the colors that we put together and define the theme with our new semantic tokens. Since the variables exist for both light and dark modes there shouldn't be a situation where we need to have separate classes for each.